decentraland/unity-renderer
# feat: disabled friends ui by a feature flag

## What does this PR change?

`Fixes #3914 `

Enables or disables the friends UI depending on a feature flag: `explorer-friends_enabled`.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/friends-feature-flag
2. None of the UI should be referring to the friends feature
3. Go to: https://play.decentraland.org/?renderer-branch=feat/friends-feature-flag
4. The friends UI should be enabled as usual

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
